### ISSUE TITLE
Unable to query sponsors for all challenges (500 error)

### ISSUE THREAD

**Issue by rrchai** [Sage-Bionetworks/rocc-service#197]
Not all challenges can query the sponsors data, take `http://localhost:4200/dream/smc-rna` as an example:

<img width="1181" alt="Screen Shot 2021-11-10 at 8 03 05 AM" src="https://user-images.githubusercontent.com/73901500/141121124-e3c0f4d0-c7ec-4698-aa6b-c70e430287c5.png">

**Comment by tschaffter** [comment]
I confirm that I can reproduce the issue. It does not appear to be an auth issue because I'm authentified and I can get a challenge readme successfully.

**Comment by tschaffter** [comment]
@rrchai I found the issue. The schemas says that the length of an account login must be <= 25 chars. However the current seed include sponsor with login value longer than this limit.

```
genome-canada
international-cancer-genome-consortium
national-cancer-institute
natural-sciences-and-engineering-research-council
ontario-institute-for-cancer-research
prostate-cancer-canada
```

On Monday, we discussed that we should not create systematically user account for every organizers. This change also involves not setting `ChallengeOrganizer.login` when creating the `ChallengeOrganizers`.

The same goes for `ChallengeSponsors`. We should not create an org account for every sponsor. This change also involves not setting `ChallengeSponsor.login` when creating the `ChallengeSponsors`.

Now tracking this bug [here](https://github.com/Sage-Bionetworks/rocc-db-client/issues/32).
